README.md

```diff
@@ -120,19 +120,22 @@ Here's an overview of the architecture:
 	# (optional) require authentication for viewing content
 	scoold.is_default_space_public = false
 	```
-3. Start Scoold with `java -jar -Dconfig.file=./application.conf scoold-*.jar`
+3. Start Scoold with `java -jar -Dconfig.file=./application.conf scoold-*.jar`.
 4. Open [localhost:8000/signin/register](http://localhost:8000/signin/register) and
-register a new account with same email you put in the configuration
+register a new account with same email you put in the configuration.
 
-If you want to login with a social account, first you *need* to create a developer app with
+To login with a social account, you must create a developer app with
 [Facebook](https://developers.facebook.com),
-[Google](https://console.developers.google.com) or **any other identity provider** that you wish to use.
-This isn't necessary if you're planning to login with LDAP, SAML or with email and password.
+[Google](https://console.developers.google.com) or any other identity provider.
+This is not required when you log in with LDAP, SAML or email/password.
+
+misplaced
 Save the obtained API keys in `application.conf`, as shown below.
```

Comment:
I was not sure where to put this line, feels diconnected from the content around it.

```suggestion
Save the obtained API keys in `application.conf`, as shown below.
```